templates/example_cell_clustering.ipynb

```diff
@@ -39,12 +39,15 @@
     "\n",
```

Comment:
<p>Such a small comment, but I think line 57 in cell_consensus_cluster.R should be deleted (the next line down is probably the one you want to keep).</p><p> </p><p>Line that says:</p><pre class="ql-syntax" spellcheck="false">clusterAvgsScale &lt;- clusterAvgs[,clusterCols]
</pre><p> </p><p>That file wasn't changed in this pull request so can't comment on the line directly, but figured I would just add it here since I'm looking at it now.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/ark-analysis/pull/497/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

Reply:
<p>Yup, this was a line I forgot to remove in the previous PR.</p>